openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -429,9 +429,17 @@ def process_resources(self, instance, staging_dir):
                 # node doesn't have color space attribute
                 color_space = "Raw"
             else:
-                if files_metadata[source]["color_space"] == "Raw":
+                # get all the resolved files in Maya File Path Editor 
+                src = files_metadata.get(source)
+                if src:
+                    if files_metadata[source]["color_space"] == "Raw":
+                        # set color space to raw if we linearized it
+                        color_space = "Raw"
+                else:
+                    # if the files are unresolved
+                    if files_metadata[filepath]["color_space"] == "Raw":
```

Comment:
⚠️ 

I'd say this is broken. Isn't this `filepath` variable by pure luck the "last iteration" in the `files_metadata` for loop and thus likely not the filepath for the `resources` being iterated here?